src/client/gui/lib/catalogue/launch_form.dart

```diff
@@ -83,6 +84,69 @@ class _LaunchFormState extends ConsumerState<LaunchForm> {
       width: 360,
     );
 
+    final zonesAsync = ref.watch(zonesProvider);
+    final zoneDropdown = FormField<String>(
+      initialValue: 'auto',
+      onSaved: (value) =>
+          launchRequest.zone = value == 'auto' ? '' : value ?? '',
+      builder: (field) {
+        return Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            Row(
+              children: [
+                const Text('Pseudo zone'),
+                const SizedBox(width: 4),
+                Tooltip(
+                  message:
+                      'Pseudo zones simulate availability zones in public clouds\nand allow for testing resilience against real-world incidents',
+                  child: Icon(Icons.info_outline,
+                      size: 16, color: Colors.blue[700]),
+                ),
+              ],
+            ),
+            const SizedBox(height: 8),
+            zonesAsync.when(
+              loading: () => const SizedBox(
+                width: 20,
+                height: 20,
+                child: CircularProgressIndicator(),
+              ),
+              error: (_, __) => const Text('Error loading zones'),
+              data: (zones) {
+                final availableZones = zones.where((z) => z.available).toList();
+                final hasAvailableZones = availableZones.isNotEmpty;
+
+                return Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: [
+                    Dropdown<String>(
+                      value: field.value,
+                      onChanged: field.didChange,
+                      items: {
+                        'auto': 'Auto (assign automatically)',
+                        for (final zone in availableZones) ...<String, String>{
+                          zone.name: 'Zone ${zone.name}',
+                        },
+                      },
+                    ),
+                    if (!hasAvailableZones)
+                      const Padding(
+                        padding: EdgeInsets.only(top: 8),
+                        child: Text(
+                          'Cannot launch new instances - all zones are unavailable',
+                          style: TextStyle(color: Colors.red),
+                        ),
+                      ),
```

Comment:
There is a much cleaner way to achieve this. Our `Dropdown` contains an `InputDecoration` which has an `errorText` property. You could add `final String? errorText` to our `Dropdown` and forward it to the `InputDecoration`. Then, you don't need to do all of this layout manually, you can just do something like
```dart
Dropdown<String>(
  ...
  errorText: availableZones.isEmpty ? 'All zones are unavailable' : null,
  ...
)
```
This will also allow you to get rid of the parent `Column`